pkg/apis/apis/v1alpha1/permissionclaims/labels.go

```diff
@@ -39,6 +39,20 @@ func ToLabelKeyAndValue(exportClusterName logicalcluster.Name, exportName string
 	return apisv1alpha1.APIExportPermissionClaimLabelPrefix + exportHash, claimHash, nil
 }
 
+// ToReflexiveAPIBindingLabelKeyAndValue returns label key and value that is set (as fallback for filtering)
+// on APIBindings that point to the given APIExport and the binding has not accepted a claim to it.
+func ToReflexiveAPIBindingLabelKeyAndValue(exportClusterName logicalcluster.Name, exportName string) (string, string) {
+	claimHash := toBase62([28]byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 1, 2, 3, 4, 5, 6, 7})
+	exportHash := toBase62(sha256.Sum224([]byte(exportClusterName.Join(exportName).String())))
```

Comment:
can we elaborate in a comment why we use base62 and Sum224?

Reply:
oh, we have that everywhere in the code-base. It's our compromise for a reasonable size, and base62 because it is the maximum you get into labels.